README.md

```diff
@@ -228,9 +102,8 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
- **Hi @henokkhm, your readme looks good but you forgot to edit this line or remove it as you have done with the others. Please remove before merging :pray:**